typings/index.d.ts

```diff
@@ -965,11 +965,11 @@ declare module 'klasa' {
 		public max: number;
 		public regex: RegExp;
 
-		private static resolveLimit(limit: string, type: string): number;
+		private resolveLimit(limit: string, limitType: "min" | "max"): number;
```

Comment:
Please let me know if this is not a valid TS type

Reply:
just `string`